### ISSUE TITLE
Ores are taken from inventory but the forges remain empty.

### ISSUE THREAD

**Issue by Xegrimoth** [BruceOfTheBow/BruceComfyMods#12]
What the titles says pretty much, I have no mods that  should interfere with this. This happened only after the Ashlands update. Hope this helps!

**Comment by blzzua** [comment]
I had same problem. 
after the Ashlands update.

also in Player.log i recieve these records:

Putting multiple wood in charcoad 
```
05/15/2024 19:28:11: Activating default element Continue
[Warning: Unity Log] 05/15/2024 19:28:16: Failed to find rpc method 44531401

05/15/2024 19:28:16: Failed to find rpc method 44531401
[Warning: Unity Log] 05/15/2024 19:28:16: Failed to find rpc method 44531401

```
05/15/2024 19:28:16: Failed to find rpc method 44531401
[Info   : Unity Log] 05/15/2024 19:28:19: Activating default element Continue


putting multiple wood in hearth
```
05/15/2024 19:30:05: Setting selected recipe 2
[Warning: Unity Log] 05/15/2024 19:30:12: Failed to find rpc method -2140177757

05/15/2024 19:30:12: Failed to find rpc method -2140177757
[Warning: Unity Log] 05/15/2024 19:30:12: Failed to find rpc method -2140177757


```

**Comment by BruceOfTheBow** [comment]
I will take a look and try to update by this weekend. Please uninstall/disable the mod for now. I think a lot of the RPC calls changed in the Ashlands update and I've not had time to update any of my mods that broke as a result. Thanks for the detailed reports, they help a lot!

**Comment by BruceOfTheBow** [comment]
This should be fixed now with this commit https://github.com/BruceOfTheBow/BruceComfyMods/commit/5dee1e82148ce8dbb36ea5fb11375e6ecb9845dd. Released on Thunderstore as ComfyAddAllFuel v1.9.0.
